src/Message/EnvelopeTrait.php

```diff
@@ -8,6 +8,18 @@ trait EnvelopeTrait
 {
     private MessageInterface $message;
 
+    /**
+     * A mirror of {@see MessageInterface::fromData()}
```

Comment:
I'd duplicate a description here. It's not really useful to go to another method phpdoc to find out what it is for.